Soccar [Game-Server]/src/nl/soccar/gameserver/message/PlayerJoinedSessionMessage.java

```diff
@@ -2,6 +2,8 @@
 
 import nl.soccar.gameserver.message.handler.PlayerJoinedSessionMessageHandler;
 import nl.soccar.library.Player;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this unused import 'nl.soccar.library.Player'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://194.135.91.67:9000/coding_rules#rule_key=squid%3AUselessImportCheck)